src/const.js

```diff
@@ -113,3 +113,8 @@ export const VALID_MANIFEST_VERSION = 2;
 // The max file size in MB that the
 // io classes will open as strings or streams.
 export const MAX_FILE_SIZE_MB = 100;
+
+export const HIDDEN_FILE_REGEX = /^__MACOSX\//;
+export const FLAGGED_FILE_REGEX = /Thumbs.db$|.DS_STORE$|.orig$|.old$|\~$/;
```

Comment:
I don't know if .DS_STORE would ever be lower case? It might make sense just do (t|T) or something.